stdlib/public/ClangOverlays/float.swift.gyb

```diff
@@ -13,6 +13,8 @@
 @_exported import _Builtin_float
 
 @available(swift, deprecated: 3.0, message: "Please use 'T.radix' to get the radix of a FloatingPoint type 'T'.")
+@available(macOS 10.9, iOS 7.0, watchOS 2.0, tvOS 9.0, *)
+@_originallyDefinedIn(module: "Darwin", macOS 9999, iOS 9999, watchOS 9999, tvOS 9999)
```

Comment:
Would it make sense to use the macros defined in `utils/availability-macros.def`?

```suggestion
@available(SwiftStdlib 5.0)
@_originallyDefinedIn(module: "Darwin", SwiftStdlib 6.0)
```

Reply:
And also they don't cover anything prior to Swift 5